Source/Search/Mixins/MultiSearchMixin.lua

```diff
@@ -0,0 +1,51 @@
+AuctionatorMultiSearchMixin = {}
+
+function AuctionatorMultiSearchMixin:InitSearch(completionCallback, incrementCallback)
```

Comment:
The results format for an undercut check search will be different to that for a multisearch, otherwise mixing it in should be possible, to get booleans for undercut

Reply:
Yeah the consumer gets to decide how to use the results format in the `completionCallback`; the provider and multisearch are just aggregating subsequent calls.